lib/services/auth.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const responses = require('./responses'),
+  _get = require('lodash/get'),
+  AUTH_LEVELS_MAP = {
+    ADMIN: 'admin',
+    WRITE: 'write'
+  };
+
+/**
+ * Gets the user auth level and check it against the
+ * required auth level for a route. Send an error
+ * if the user doesn't have permissions.
+ * @param {String} requiredLevel
+ * @return {Function}
+ */
+function withAuthLevel(requiredLevel) {
+  return (req, res, next) => {
+    if (checkAuthLevel(_get(req, 'user.auth', ''), requiredLevel)) {
```

Comment:
I'm pretty confident that we don't need to port over authentication levels since anyone who can login should be able to schedule/unschedule a page. Instead I would just check if the user is authenticated like `amphora-search` currently does: https://github.com/clay/amphora-search/blob/master/lib/routes/_search.js#L61-L65